.drone.yml

```diff
@@ -0,0 +1,40 @@
+build:
+  image: ubuntu:xenial
```

Comment:
Is there a way to have a build matrix of different base images.  It would be nice to test on 14.04 too (the previous LTS release?)